draft-ietf-quic-transport.md

```diff
@@ -2648,6 +2650,10 @@ closes a stream SHOULD be discarded.  An endpoint MAY choose to limit the period
 over which it ignores frames and treat frames that arrive after this time as
 being in error.
 
+An endpoint may receive a RST_STREAM after a FIN if the peer resets the stream
+after sending a FIN on it. In this case, the endpoint MAY discard any data that
+it already received on that stream.
```

Comment:
How about adding a sentence that the endpoing MAY close the connection if a stream is reset with a different offset from the previously received fin.

Reply:
I would think should